frontend/public/module/k8s/pods.ts

```diff
@@ -248,7 +248,7 @@ export const podPhase = (pod: PodKind): PodPhase => {
         : `Init:ExitCode:${terminated.exitCode}`;
     } else if (waiting && waiting.reason && waiting.reason !== 'PodInitializing') {
       phase = `Init:${waiting.reason}`;
-    } else {
+    } else if (!running && !ready) {
```

Comment:
IIUC, this would cause the loop over init containers to be terminated after encountering the first sidecar, which is probably not what we want.